CHANGELOG.rst

```diff
@@ -39,6 +39,13 @@ Fixed
 * Fixed generation of `st2.conf.sample` to show correct syntax for `[sensorcontainer].partition_provider` (space separated `key:value` pairs). #5710
   Contributed by @cognifloyd
 
+* A new output schema using full JSON schema was introduced and secrets previously masked using
+  the legacy output schema are now being displayed as plain text. To prevent security relative
+  issues, add backward compatibility to secret masking. Full output schema validation will need
+  to be migrated to the new schema.
```

Comment:
Can we refactor the messaging here a bit? 
```suggestion
* Add backward compatibility to secret masking introduced in #5319 to prevent security-relative issues. Full output schema validation will need
  to be migrated to the new schema. #5783
```
It sounds like there was an issue in the previous version of stackstorm.


In fact, this patch is added on top of #5319, which wasn't released yet and there was no harm made to the production versions yet.

It's great to have this issue caught early though!

Reply:
Done